<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="CentralGridLayoutWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionSave_As"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionSelect"/>
   <addaction name="actionCrop"/>
   <addaction name="actionRegion_Of_Interest"/>
   <addaction name="actionContrast"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Umamaheswaran/Desktop/document-open-7.ico</normaloff>C:/Users/Umamaheswaran/Desktop/document-open-7.ico</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Umamaheswaran/Desktop/edit-undo-7.ico</normaloff>C:/Users/Umamaheswaran/Desktop/edit-undo-7.ico</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Umamaheswaran/Desktop/edit-redo-7.ico</normaloff>C:/Users/Umamaheswaran/Desktop/edit-redo-7.ico</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Umamaheswaran/Desktop/document-save-as-5.ico</normaloff>C:/Users/Umamaheswaran/Desktop/document-save-as-5.ico</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Umamaheswaran/Desktop/zoom-in-3.ico</normaloff>C:/Users/Umamaheswaran/Desktop/zoom-in-3.ico</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Umamaheswaran/Desktop/zoom-out-3.ico</normaloff>C:/Users/Umamaheswaran/Desktop/zoom-out-3.ico</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Umamaheswaran/Desktop/transform-crop-and-resize.ico</normaloff>C:/Users/Umamaheswaran/Desktop/transform-crop-and-resize.ico</iconset>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
   <property name="toolTip">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Umamaheswaran/Desktop/select-rectangular.ico</normaloff>C:/Users/Umamaheswaran/Desktop/select-rectangular.ico</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select</string>
   </property>
  </action>
  <action name="actionRegion_Of_Interest">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Umamaheswaran/Desktop/select.ico</normaloff>C:/Users/Umamaheswaran/Desktop/select.ico</iconset>
   </property>
   <property name="text">
    <string>Region Of Interest</string>
   </property>
   <property name="toolTip">
    <string>Region Of Interest</string>
   </property>
  </action>
  <action name="actionContrast">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Umamaheswaran/Desktop/Contrast.ico</normaloff>C:/Users/Umamaheswaran/Desktop/Contrast.ico</iconset>
   </property>
   <property name="text">
    <string>Contrast</string>
   </property>
   <property name="toolTip">
    <string>Auto Contrast</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CentralGridLayoutWidget</class>
   <extends>QWidget</extends>
   <header>centralgridlayoutwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
